<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>596</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1188</width>
    <height>600</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>InfoPro Setup</string>
  </property>
  <property name="windowIcon">
   <iconset theme="application-x-executable">
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="menu1">
     <attribute name="title">
      <string>Základní nastavení</string>
     </attribute>
     <widget class="QTextBrowser" name="ipShow">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>10</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Síťová rozhraní</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="infoproURL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Zde zadejte adresu InfoPro obrazovky...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nastavitButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Nastavit adresu</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QWebEngineView" name="infoproNahled">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>901</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Náhled webu</string>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
     <widget class="QLabel" name="labelSit">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>10</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Síťová rozhraní online:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="spustitButton">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>170</y>
        <width>190</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Spustit InfoPro</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>28</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="displejButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>340</y>
        <width>150</width>
        <height>70</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Nastavení zobrazení</string>
      </property>
      <property name="text">
       <string>Displej</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="zvukButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>420</y>
        <width>150</width>
        <height>70</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Nastavení zvuku</string>
      </property>
      <property name="text">
       <string>Zvuk</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sitButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>260</y>
        <width>150</width>
        <height>70</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Nastavení sítě</string>
      </property>
      <property name="text">
       <string>Síť</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="rebootButton">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>510</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Restart zařízení</string>
      </property>
      <property name="text">
       <string>Restart</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="shutdownButton">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>510</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Vypnutí zařízení</string>
      </property>
      <property name="text">
       <string>Vypnout</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menu2">
     <attribute name="title">
      <string>Servis</string>
     </attribute>
     <widget class="QPushButton" name="terminalbutton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Spustí emulátor terminálu</string>
      </property>
      <property name="text">
       <string>Terminál</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hostsButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>470</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>/etc/hosts</string>
      </property>
     </widget>
     <widget class="QPushButton" name="htopButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Správce spuštěných úloh - htop</string>
      </property>
      <property name="text">
       <string>Správce úloh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="terminalRootButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Spustí emulátor terminálu s oprávněním ROOT</string>
      </property>
      <property name="text">
       <string>Terminál (ROOT)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hostnameButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>470</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>/etc/hostname</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sensorsButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Sledování teplotních senzorů zařízení</string>
      </property>
      <property name="text">
       <string>Sledování teploty</string>
      </property>
     </widget>
     <widget class="QPushButton" name="crontabButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>470</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>/etc/crontab</string>
      </property>
     </widget>
     <widget class="QPushButton" name="firewallButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>270</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Nastavení místního Firewallu</string>
      </property>
      <property name="text">
       <string>Firewall</string>
      </property>
     </widget>
     <widget class="QPushButton" name="systemctlButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>130</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Stav služeb SSH a VNC</string>
      </property>
      <property name="text">
       <string>Stav SSH a VNC</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>311</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ruční úprava konfiguračních souborů:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="infoproAutorunButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>infopro_autorun</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>450</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>8</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>*plánovač úloh - restartování</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>450</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>8</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>*skript, který spouští prohlížeč</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>infoproURL</tabstop>
  <tabstop>nastavitButton</tabstop>
  <tabstop>spustitButton</tabstop>
  <tabstop>sitButton</tabstop>
  <tabstop>displejButton</tabstop>
  <tabstop>zvukButton</tabstop>
  <tabstop>rebootButton</tabstop>
  <tabstop>shutdownButton</tabstop>
  <tabstop>terminalbutton</tabstop>
  <tabstop>terminalRootButton</tabstop>
  <tabstop>htopButton</tabstop>
  <tabstop>sensorsButton</tabstop>
  <tabstop>systemctlButton</tabstop>
  <tabstop>firewallButton</tabstop>
  <tabstop>infoproAutorunButton</tabstop>
  <tabstop>hostsButton</tabstop>
  <tabstop>hostnameButton</tabstop>
  <tabstop>crontabButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
